lifetracker-api/lifetracker-schema.sql

```diff
@@ -0,0 +1,43 @@
+CREATE TABLE users (
+    id SERIAL PRIMARY KEY,
+    email VARCHAR(255) NOT NULL UNIQUE CHECK(POSITION('@' IN email)>1),
+    username VARCHAR(255) NOT NULL,
+    firstname VARCHAR(255) NOT NULL,
+    lastname VARCHAR(255) NOT NULL,
```

Comment:
best practice to use snake case for column name (e.g. `first_name` and `last_name`)